x-pack/plugins/apm/public/components/app/error_group_details/error_sampler/error_sample_detail.tsx

```diff
@@ -69,10 +76,17 @@ function getCurrentTab(
   return selectedTab ? selectedTab : first(tabs) || {};
```

Comment:
[:nit] since we are returning selectedTab after checking it exists maybe we can just do something like: 
```
return selectedTab ?? (first(tabs) || {});
```